x-pack/legacy/plugins/siem/public/components/ml_popover/types.ts

```diff
@@ -101,6 +102,7 @@ export interface MlSetupArgs {
  * Representation of an ML Job as returned from the `ml/jobs/jobs_summary` API
  */
 export interface JobSummary {
+  auditMessage?: AuditMessageBase;
```

Comment:
Nice that we can use the explicit ML types! ++ 🎉

Reply:
I was a little wary of this one... I'm getting no linter complaints, and that file _is_ just types, so I think we should be fine.